rustls-platform-verifier/src/lib.rs

```diff
@@ -1,4 +1,4 @@
-#![cfg_attr(docsrs, feature(doc_cfg))]
+#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
```

Comment:
Are we sure this isn't going to break again? This was [removed before](https://github.com/rustls/rustls-platform-verifier/pull/57/files#diff-8660cd1844f655870a5550ff30da2ac758384062af24800f5296599e69b95f27R1) in a previous PR of mine because it caused the wrong feature gates to appear on `docs.rs`.

Reply:
Forgot about that :+1: 

I backed out the `doc_auto_cfg` change and switched to using stable rust for the documentation task.